source/application/SGM/AMDSCentralServerSGMPV.cpp

```diff
@@ -144,22 +145,44 @@ void AMDSCentralServerSGMPV::fillConfigurationCommandForClientRequest(const QStr
 void AMDSCentralServerSGMPV::processClientDataRequest(AMDSClientRequest *clientRequest)
 {
 	// make sure that we only continue AMDS PV data request when the PV is enabled
-	QString bufferName = "";
 	bool continueProcessDataRequest = false;
 	AMDSClientDataRequest *clientDataRequest = qobject_cast<AMDSClientDataRequest*>(clientRequest);
 	if(clientDataRequest){
-		bufferName = clientDataRequest->bufferName();
-		AMDSPVConfigurationMap *pvConfiguration = pvConfigurationMaps_.value(bufferName);
-		if (pvConfiguration && pvConfiguration->enabled()) {
-			continueProcessDataRequest = true;
+		if (clientDataRequest->isContinuousMessage()) {
+			AMDSClientContinuousDataRequest *continuousDataRequest = qobject_cast<AMDSClientContinuousDataRequest *>(clientRequest);
+			QStringList bufferNames;
```

Comment:
In general, it would be better to just set the bufferNames member (`QStringList bufferNames = continuousDataRequest->bufferNames();`) rather than using append explicitly like this.